flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/CalcTestPrograms.java

```diff
@@ -36,7 +36,7 @@
  */
 public class CalcTestPrograms {
 
-    static final TableTestProgram SIMPLE_CALC =
+    public static final TableTestProgram SIMPLE_CALC =
```

Comment:
The programs are still under `/exec/stream`. Move them up or better into a `common` package

Reply:
Moved.